apps/escrow/frontend/README.md

```diff
@@ -0,0 +1,67 @@
+<h1>⚡️ Fuels UI - Example Applicatiton</h1>
+
+<h2>📝&nbsp; Table of Content</h2>
+
+- [🙋🏻&nbsp; Getting Started](#-getting-started)
+- [🛠&nbsp; Tools used](#-tools-used)
+- [🗃&nbsp; Systems Based Architecture](#-systems-based-architecture)
+- [🛤&nbsp; Routes](#-routes)
+- [📜&nbsp; License](#-license)
+
+---
+
+## 🙋🏻&nbsp; Getting Started
+
+First, run the development server:
```

Comment:
This should first mention that you need to go through the installation process in the root of the repo `CONTRIBUTIONS.md` file